lib/internal/modules/cjs/loader.js

```diff
@@ -95,12 +95,14 @@ const {
 const { validateString } = require('internal/validators');
 const pendingDeprecation = getOptionValue('--pending-deprecation');
 
+const originalModuleExports = new Map();
```

Comment:
```suggestion
const originalModuleExports = new SafeMap();
```

Reply:
nice catch, it actually needs to be a WeakMap too

Reply:
In that case we'd need a SafeWeakMap if it doesn't already exist